cleanlab/internal/constants.py

```diff
@@ -21,3 +21,25 @@
     2 * FLOATING_POINT_COMPARISON
 )  # lower bound imposed to clip confident thresholds from below, has to be larger than floating point comparison
 TINY_VALUE = 1e-100  # very tiny value for clipping
+
+
+# Object Detection Constants
+EUC_FACTOR = 0.1  # factor to control magnitude of euclidian distance
```

Comment:
standardize the formatting of the comments, some say param of objectlab, some don't.

Reply:
euclidian -> euclidean

Reply:
more explanation in english. 

This distance affects the measured similarity between predicted vs annotated boxes, and increasing this factor makes all of the similarities more homogeneous. 

(or is it less homogeneous?)